crates/polars-core/src/chunked_array/comparison/mod.rs

```diff
@@ -801,6 +804,7 @@ impl ChunkCompareEq<&StructChunked> for StructChunked {
             |l, r| l.equal(r).unwrap(),
```

Comment:
Actually, this is not what this PR is trying to solve, but we made a policy that nested equality is always eq_missing. Maybe, this should changed here too.

Reply:
Make sense, thanks.

Reply:
I think not all types support `eq_missing`, so the code reaches `panic`.
Maybe it is better to get fixed in separate PR?